src/jni_core/src/jvm_runtime.rs

```diff
@@ -134,3 +136,35 @@ pub fn register_native_method_for_jvm(jvm: &JavaVM) -> Result<(), jni::errors::E
     tracing::info!("register native methods for jvm successfully");
     Ok(())
 }
+
+pub fn load_jvm_memory_stats() -> (usize, usize) {
+    let mut env = JVM.as_ref().unwrap().attach_current_thread().unwrap();
```

Comment:
If the JVM is not loaded before, will this trigger JVM to load?

Reply:
Yes. It will, so I set the initial heap memory of the JVM to be 16M.

Reply:
This sounds not good. :( I have no idea why `LazyLock` does not expose its `get() -> Option<_>` method. Maybe we can use a global atomic value as an indicator? cc @wangrunji0408